Minimal examples changes - Uricorn (#87)
wix/headless-templates: nextjs/minimal-examples/README.md

### Code Sandbox

> 1. Fork the CodeSandbox project by click **Fork** in the top right corner, after signing in.

Reviewer: Can you add a link to the CodeSandbox while you're at it?

> 1. Fork the CodeSandbox project by click **Fork** in the top right corner, after signing in.

Reviewer: ```suggestion
1. To fork the CodeSandbox project, sign in and click **Fork** in the top right corner.
```

### Copy the project to your local workspace

> 1. Clone the [github repo](https://github.com/wix/headless-templates/tree/main/nextjs/minimal-examples).

Reviewer: ```suggestion
1. Clone the [Headless Templates repo](https://github.com/wix/headless-templates/tree/main/nextjs/minimal-examples).
```
